patroni/ha.py

```diff
@@ -1359,9 +1363,16 @@ def _run_cycle(self):
 
                 if self.sysid_valid(self.cluster.initialize):
                     if self.cluster.initialize != data_sysid:
-                        logger.fatal("system ID mismatch, node %s belongs to a different cluster: %s != %s",
-                                     self.state_handler.name, self.cluster.initialize, data_sysid)
-                        sys.exit(1)
+                        if self.is_paused():
+                            logger.warning('system ID has changed while in paused mode. Patroni will exit when resuming'
+                                           ' unless system ID is reset: %s != %s', self.cluster.initialize, data_sysid)
```

Comment:
Should system id or  the `/initialize` key be reset ? Is resetting the `/initialize` in DCS to the new system ID sufficient to prevent Patroni from exiting on resume ?

Reply:
I think when there is another node, resetting the `/initialize` key would break that one

Reply:
Patroni should not reset or change the valid initialize key. It is used as a safety measure to prevent the wrong node from joining the current cluster.